README.md

```diff
@@ -1,77 +1,122 @@
+> Welcome to Team Apollo's sales tracker revolving around the creation of an ETL pipeline via the use of Bandcamp's API and webscraping. The data is then used to create three different outputs which will provide various information on artists and genres regarding their sales and popularity both recently and over time. These are: **dashboard**, **email notifications**, and **PDFs**. This results in our **tracker** which is provided to consumers as a service.
 
 ### 👩‍💼 Stakeholder Requirements
-- Obtain latest sales data from Bandcamp on artists/genre
-- Automatically alert, notify, and/or send PDFs to subscribers regarding this data.
+- Obtain **latest** sales data from Bandcamp on artists/genre
+- Automatically **alert**, **notify**, and/or **send** PDFs to subscribers regarding this data.
 
 ### 🎯 Deliverables
-- A full data pipeline, hosted in the cloud.
-- A long-term storage solution for the data extracted from Bandcamp.
-- Some form of visualisation of the data.
+- A **full data pipeline**, hosted in the cloud.
+- A **long-term storage solution** for the data extracted from Bandcamp.
+- **Visualisation** of the sales data.
 
 
 ## ✏️ Design
 
 ### 📏 Entity-Relationship Diagram
 ![ERD Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/ERD.png)
+
+This diagram explicitly shows how the data extracted from the API is stored in the database. We have decided to use a **3NF RD** for our database due to the requirements of the tracker. As we care about various sources of data such as countries, tags, and artists, other types of schema such as STAR Schema would not be suitable for our aims as we need to query multiple tables. 
+
 ### 📐 Architecture Diagram
 ![Architecture Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/Architecture_Diagram.png)
 
+This is a diagram that shows how every different AWS services work and interact with each other in order to complete the requirements of the tracker.
+
+#### **IMPORTANT**
+ >Information regarding each section of the AD will not be found here. Instead, **clicking on** the bullet points below will send you to the relative **README** located in the **directories** of the section that you would like to look at. There you can find out all the information that you need.
+
+
+- [**ETL Pipeline**](./pipeline/README.md)
+
+- [**RDS**](./schema/README.md)
+
+- [**PDF Report**](./pdf_report/README.md)
+
+- [**Notifications**](./notifications/README.md)
+
+- [**Dashboard**](./dashboard/README.md)
+
+
+
+
 ## ✅ Getting Started
 
 ### 💿 Installations
-The following languages/softwares are required for this project. Things assigned as optional are only required if you desire to host this system on the cloud.
+The following languages/softwares are **required** for this project. Things assigned as **optional** are only required if you desire to host this tracker on the cloud.
 - Python
 - Bash
 - Docker (Optional)
```

Comment:
The service as a whole wouldn't be properly operational without deployment to the cloud, I don't think you need to label docker as optional.

Reply:
Fairs.